src/sqlite3pp.h

```diff
@@ -92,13 +101,19 @@ namespace sqlite3pp
 
     int set_busy_timeout(int ms);
 
+    int load( const char *filename );
```

Comment:
Let's not add these load and save here. I prefer these done outside of the sqlite3pp class.